### ISSUE TITLE
Improve UI for per-row saving in CRUD table

### ISSUE THREAD

**Issue by rossng** [clowdr-app/clowdr#224]
At the moment, the new CRUD table implements per-row autosaving (#182). But it's not clear in the UI that this is happening.

We should implement some indicator that the row data is changed locally but not saved yet. Perhaps even by displaying a manual save button that the user can optionally choose to click.

**Comment by hadley0828** [comment]
Hi, I have made some change to the input value and make it red when the value is changed but not saved. The result is shown as below.

https://user-images.githubusercontent.com/18096538/132252959-e4d00746-8f72-4d19-9335-78070cce5cd4.mov

For the detailed coding part, I commented two lines of code in CRUDTable2.tsx file in order to achieve this function. And I am wondering whether this will cause some problems in other pages.
![image](https://user-images.githubusercontent.com/18096538/132253113-fa155fdd-8869-496b-a6bf-01810e383403.png)

Or is there some suggestions to implement it without changing this part? Looking forward to hear from you guys!
